change/@azure-msal-node-a37dcd93-77ce-4c8e-bdfb-6437145819be.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "ExtraTokenParameters are now sent to the /token endpoint when using acquireTokenByClientCredential #5573",
```

Comment:
```suggestion
  "comment": "tokenQueryParameters are now sent to the /token endpoint when using acquireTokenByClientCredential #5573",
```

Reply:
also, seems to me the only change to msal-node is the added comment -should this actually be a patch?